src/components/GroupsTable/GroupsTable.js

```diff
@@ -0,0 +1,237 @@
+/* eslint-disable camelcase */
+import {
+    Pagination,
+    PaginationVariant,
+    SearchInput
+} from '@patternfly/react-core';
+import {
+    cellWidth,
+    sortable,
+    Table,
+    TableBody,
+    TableHeader,
+    TableVariant
+} from '@patternfly/react-table';
+import {
+    DateFormat,
+    ErrorState,
+    PrimaryToolbar
+} from '@redhat-cloud-services/frontend-components';
+import debounce from 'lodash/debounce';
+import upperCase from 'lodash/upperCase';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { TABLE_DEFAULT_PAGINATION } from '../../constants';
+import { fetchGroups } from '../../store/inventory-actions';
+import { generateLoadingRows } from '../InventoryTable/helpers';
+import NoEntitiesFound from '../InventoryTable/NoEntitiesFound';
+
+const GROUPS_TABLE_INITIAL_STATE = {
```

Comment:
How about extracting assets into separate file?

Reply:
i'd like to keep it in the same file since these constants are tied to the component's state only

Reply:
Hmm, maybe there will be a new component that can re-use them? But, not a big deal, this is minor :)